test/remote/README.md

```diff
@@ -128,7 +128,7 @@ Run the tests
 In another shell, do a port-forward to the API server's service:
 
 ```shell
-kubectl -n argocd-e2e port-forward svc/argocd-server 443:4443
+kubectl -n argocd-e2e port-forward svc/argocd-server 4443:443
```

Comment:
Any reason to update this?

Reply:
Uhm, yes. The notation is `local:remote`. Previously the export in the doc says:
```
export ARGOCD_SERVER=127.0.0.1:4443
```
Expecting the argocd-server to be on 4443/tcp. If we're forwarding 443 locally, then the TCP syn sent to 4443/tcp will get an RST, meaning the connection is refused.

From the other side, if we connect to 443/tcp locally, then it will be forward to port 4443/tcp on the pod. The argocd-server service is defined as:
```
---
apiVersion: v1
kind: Service
metadata:
  labels:
    app.kubernetes.io/component: server
    app.kubernetes.io/name: argocd-server
    app.kubernetes.io/part-of: argocd
  name: argocd-server
spec:
  ports:
  - name: http
    port: 80
    protocol: TCP
    targetPort: 8080
  - name: https
    port: 443
    protocol: TCP
    targetPort: 8080
  selector:
    app.kubernetes.io/name: argocd-server
```

So, if we're forwarding the local 443 to remote 4443, it will get refused, because the service listens on 443 remotely, and not on 4443.